apps/nsqd/options.go

```diff
@@ -148,7 +148,7 @@ func nsqdFlagSet(opts *nsqd.Options) *flag.FlagSet {
 	flagSet.String("data-path", opts.DataPath, "path to store disk-backed messages")
 	flagSet.Int64("mem-queue-size", opts.MemQueueSize, "number of messages to keep in memory (per topic/channel)")
 	flagSet.Int64("max-bytes-per-file", opts.MaxBytesPerFile, "number of bytes per diskqueue file before rolling")
-	flagSet.Int64("max-depth", opts.MaxDepth, "number of messages per diskqueue before queue stops accepting new messages")
+	flagSet.Int64("max-disk-space", opts.MaxDiskSpace, "max disk space of messages per diskqueue before queue starts removing old data")
```

Comment:
1. Is it bytes, Mb, Kb?
2. is it specified per topic or for whole data, or per file?
3. how it removing the old data?

Reply:
I'll rename to max-bytes-per-diskqueue or max-bytes-per-channel-and-topic ?

Reply:
also we can separate to multiple different options max-bytes-per-channel/max-bytes-per-topic